src/ui/components/Carousel/index.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+import React from 'react';
+import NukaCarousel from 'nuka-carousel';
+
+import Card from 'ui/components/Card';
+
+import './styles.scss';
+
+
+type PropTypes = {|
+  sections: Array<any>,
+|};
+
+const Carousel = ({ sections }: PropTypes) => {
+  if (!sections) {
+    throw new Error('sections are required for a Carousel component');
+  }
+
+  return (
+    <Card className="Carousel">
+      <NukaCarousel
+        autoplay={false}
+        autoplayInterval={4000}
+        cellAlign="left"
```

Comment:
I don't know what this property does but shouldn't this be RTL/LTR dependent?

Reply:
Could this be the reason why I get this?

<img width="1392" alt="screen shot 2017-10-04 at 15 28 03" src="https://user-images.githubusercontent.com/217628/31178231-aab1f7a6-a918-11e7-8215-0a02ed7dd733.png">

Reply:
Hmm... that should be tweaked. I'll have a look. The RTL is still buggy.